src/Cosmos/JsonSerialization.cpp

```diff
@@ -136,12 +138,27 @@ static json messageWithdrawReward(const Proto::Message_WithdrawDelegationReward&
     };
 }
 
+json messageWasmTransfer(const Proto::Message_WasmExecuteContractTransfer& msg) {
+    return {
+        {"type", TYPE_PREFIX_WASM_MSG_EXECUTE},
+        {"value",
+            {
+                {"sender", msg.sender_address()},
+                {"contract", msg.contract_address()},
+                {"execute_msg", wasmExecuteTransferPayload(msg)},
```

Comment:
I diffed the proto in terra fork and cosmwasm, the field name is slightly different, in this case: `msg` vs `execute_msg`, `funds` vs `coins`. this json message will only work for terra, maybe we should add a separate cpp file for terra?

Reply:
+1, se comment below

Reply:
Added name to relevant proto/method.